Makefile

```
@@ -265,6 +267,10 @@ manifests: ## Generate K8s manifests e.g. CRD, RBAC etc.
 	$(eval TMPDIR := $(shell mktemp -d))
 	$(QUIET)$(GO) run sigs.k8s.io/controller-tools/cmd/controller-gen $(CRD_OPTIONS) paths=$(CRD_PATHS) output:crd:artifacts:config="$(TMPDIR)"
 	$(QUIET)$(GO) run ./tools/crdcheck "$(TMPDIR)"
+
+	# Clean up old CRD state and start with a blank state.
+	for path in $(CRDS_CILIUM_PATHS); do rm -rf $${path} && mkdir $${path}; done
```

Some nitpick suggestions but they don't really matter, I can't find a case where even `mkdir -p` would be necessary. Just “good practices” I guess.

```suggestion
	# Clean up old CRD state and start with a blank slate.
	for path in $(CRDS_CILIUM_PATHS); do $(RM) -rf -- $${path} && mkdir -p $${path}; done
```